src/5e-SRD-Monsters.json

```diff
@@ -8810,7 +9584,19 @@
     "actions": [
       {
         "name": "Multiattack",
-        "desc": "The chuul makes two pincer attacks. If the chuul is grappling a creature, the chuul can also use its tentacles once."
+        "desc": "The chuul makes two pincer attacks. If the chuul is grappling a creature, the chuul can also use its tentacles once.",
```

Comment:
We could probably use a `note` here too? That way we can add the grapple modifier too? Maybe? I know this one is a little weird.